generate_config.py

```diff
@@ -11,6 +11,15 @@ def is_intake_key(intake_key: str) -> re.Match[str] | None:
     pattern = "^[a-zA-Z0-9]{16}"
     return re.search(pattern, intake_key)
 
+def activate_monitoring(intake_key: str):
+    to_print.append("Forwarder monitoring is active")
+    to_print.append("Intake key: " + str(item["intake_key"]))
+    to_print.append("")
+    config = template_stats.render(item)
+    filename = f"/etc/rsyslog.d/stats_{item['name'].lower()}.conf"

```

Comment:
If possible, we probably should handle the case the name contains spaces everywhere the name is used